tests/css_trailing_comma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -11,16 +11,64 @@ a {
 
 `;
 
+exports[`declaration.css 2`] = `
+a {
+    margin: $bar,;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+a {
+  margin: $bar;
+}
+
+`;
+
 exports[`list.css 1`] = `
 $colors: hotpink deepskyblue firebrick,;
+$list: (a,);
+$list: ('Helvetica', 'Arial', sans-serif,);
+$colors: (
+  "red",
+  "blue"
+);
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 $colors: hotpink deepskyblue firebrick;
+$list: (a);
+$list: (
+  "Helvetica",
+  "Arial",
+  sans-serif
+);
+$colors: (
+  "red",
+  "blue"
+);
```

Comment:
Let's add tests for nested maps and lists inside maps

Reply:
I have added the tests. is this the expected behaviour when nested [playgroundlink](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASSAbCAnAzkgAgAsIYAHASygGsCATOOM3agTwCMMBXOAgMwrY47bBTDUANAG4AOlFQYKuGIQAUAQwkBKWfMXK1AcgAScDADc4MMesMSChgIKj1GOwVzqouALS44onzauugQWHhqcgQEMiBCdLESUTEgnDyxcjpyoVACAOaRUNEwRHAAtnD4BKrJ0WVKKtW10cRw6gzYhADEdGB86mAADElFLdGQsAiNXYODAJwAbAAc7CNj0XwQpAHdSwCsYADsc2DNWmstqLhkolAFTaNjpe07BF18ACz9RxdjE-CwbpgABMHz2Hzgvxam22nTefD47AAjHslmdklpMrpsiI2tQyBAqDAfGV1GRCtFcKSMBgKS16lAfAB3Ch0EqEKBcGlQ0kAD2ZrPZBA+xzIvKh7HU-h8m0BBCRCzFUMs2GsYFcPmwRFYJTKhCRADoAMwYi4VOgULh6h5jBkCtlEQgfJaDJXNPn2oVzPZzN2PAiS6WyxpIpZ+sYqtUarU6ojWw0fJEfU3JDDqbB5OB0upUT2OgiLV3i93qfksh36+a+4v+wNwGXQENhmsRgJRjCa7W6-UGvYp0a8tMZrM2+m58tCpFI2bh+mlvMcrkYCVS+vBwjApGz6KRmwdmPd+UGhahjFYuQgCQgCBkazQXDIUDp7AQJkABXTCAfKHU5kJdEvEApRgZB+gwfwrxEAZqCsABlMgBioPJkBgbAeCvOgIDAUDXAgkAqH8VU32wdQ8lJHDwMhEAACtcF5AAhEjxDg9QKgAGSoOAKLwiAuHIPjgW4qiELwAJkBAXAwFwB8rxuIkAHVBSIZAXVkl9-HkkiyHEm5KgCSxAKEABHLhBDgYjSPIpAwLw-x6hQtCqNwJCMDgABFLhtiEq8YHUdhFIdZBgR8kiKEUO4AGEIDKKyQCgaAuKvLh-AAFT878bLgABfLKgA) ?

According to style guide trailing comma should not be showing when in single line so is there any better way of detecting if its exceeding printWidth except for looping through the node node and checking the length occupied and then decide whether to put trailing comma ?